cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml

```diff
@@ -7,26 +7,26 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  name: heapster-v14
+  name: heapster-v15
   namespace: kube-system
   labels:
     k8s-app: heapster
-    version: v14
+    version: v15
```

Comment:
Consider making `v15` and `v0.20.0-alpha11` templated parametrers (like `heapster_memory`) so we only need to change it in 1 place